popup.js

```diff
@@ -237,24 +237,24 @@ function handleSubmit( e ) {
   data[COMMMENT_ID] = commentID;
 
   $.get(GOOGLE_FORMS_URL, data)
-    .then(() => {
+    .then(function() {
       $('#submit').prop('disabled', false);
 
       showStatus('success', 'URL submitted. Thanks!');
       hideStatus(3000);
       // uncomment this line to also add the URL through the official notificaiton tool.
       // window.open(NOTIFICATION_TOOL_URL + currentURL);
     })
-    .catch((error) => {
+    .catch(function(error) {
       $('#submit').prop('disabled', false);
-      showStatus('failure', `Could not submit URL (${ error.statusText || 'Generic error' })`);
-    });
+      showStatus('failure', 'Could not submit URL (' + error.statusText || 'Generic error');
```

Comment:
The expression on the right requires parentheses: `'Could not submit URL (' + (error.statusText || 'Generic error')`